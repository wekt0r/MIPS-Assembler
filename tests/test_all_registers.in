sub $zero, $ra, $zero
add $at, $v0, $v1
slt $v0, $v1, $a0
sltu $a3, $t0, $t7
or $s0, $s7, $t8
xor $t9, $k0, $k1
and $gp, $sp, $fp
