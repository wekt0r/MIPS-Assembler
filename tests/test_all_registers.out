.text
00000000	03E00022	sub	$zero,$ra,$zero
00000004	00430820	add	$at,$v0,$v1
00000008	0064102A	slt	$v0,$v1,$a0
0000000c	010F382B	sltu	$a3,$t0,$t7
00000010	02F88025	or	$s0,$s7,$t8
00000014	035BC826	xor	$t9,$k0,$k1
00000018	03BEE024	and	$gp,$sp,$fp
